src/components/portrait/portrait-gravatar.component.js

```diff
@@ -30,12 +40,14 @@ class PortraitGravatar extends React.Component {
         onError={errorCallback}
         data-element="user-image"
         {...otherProps}
+        {...this.marginProps()}
```

Comment:
I think that we don't need this one. Without this I think all margin props will come from `...otherProps` as well

Reply:
Removed as requested. Thanks for spotting this 👍🏻